### PR TITLE
Game loop

### PR DESCRIPTION
**Pull request by Lucas-ws**
Added the functionality for each players turn allowing them to roll and move!

### CODE REVIEW

File: src/core/Utils.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    public static void ClearConsole() {
+        System.out.println(System.lineSeparator().repeat(50));
+    }
+
+    /**
+     * Check if a player is ready.
+     */
+    public static void readyCheck() {
+        System.out.print("Type 'Y' when ready...\n> ");
+        while(!scanner.hasNext("Y|y")){
```
Comment (TomasBorsje) [inline src/core/Utils.java:22]: Could maybe change this into any next input?
"Press any key to continue..." etc

**Review by TomasBorsje — COMMENTED** [review]
Looks great, need your thoughts on changing ready message. Otherwise will merge!

File: src/core/Utils.java (hunk 1 above)
Comment (Lucas-ws) [reply src/core/Utils.java]: Hi Tomas, 

please enjoy the revised changes to readyCheck.

Condolences,
Lucas

**Comment by TomasBorsje** [comment]
Looks good!

**Pull request merged**